doc/Makefile

```diff
@@ -54,12 +54,12 @@ hpp-template-corba_repository=${HPP_REPO}
 
 # Either a version tag (e.g. v4.3.0), master or devel
 HPP_VERSION=devel
-HPP_EXTRA_FLAGS=-DDISABLE_TESTS=${DISABLE_TESTS}
+HPP_EXTRA_FLAGS=-DDISABLE_TESTS=${DISABLE_TESTS} -DCMAKE_CXX_STANDARD=11
```

Comment:
Does this make any difference ? It shouldn't...

Reply:
That's what I'm currently looking at, but I had forgotten that I had an open PR here, sorry for the noise